torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -791,12 +791,21 @@ def _lower_static_weighted_ref_functional(
         else:
             raise ValueError("Lowering is not supported for op '%s'" % func_node.target)
         with model.graph.inserting_before(output_scale_node):
-            packed_weight = model.graph.create_node("call_function", prepack_op, tuple(prepack_args), {})
+            # kwargs of the func node are needed for prepack op (i.e., quantized::linear_prepack)
+            # They are not needed for compute op (i.e., quantized::linear)
+            kwargs = func_node.kwargs
+            # F.linear uses 'bias' key for bias while qlinear_prepack uses 'B' for bias
+            if func_node.target == F.linear and 'bias' in kwargs:
+                kwargs = kwargs.copy()
+                kwargs['B'] = kwargs['bias']
+                del kwargs['bias']
+            packed_weight = model.graph.create_node("call_function", prepack_op, tuple(prepack_args), kwargs)
 
         # Step 2: Replace reference pattern with the corresponding quantized op
         (q_func, q_relu_func) = STATIC_LOWER_FUNCTIONAL_MAP[func_node.target]  # type: ignore[index]
         func_node.target = q_relu_func if relu_node is not None else q_func
         func_node.args = (input_dq_node.args[0], packed_weight, output_scale_node, output_zp_node)
+        func_node.kwargs = {}
```

Comment:
nit: maybe add a comment here saying `kwargs` for func_node has been moved to `kwargs` for prepack op

Reply:
Thanks. Glad to see you back. Since this PR is already merged, I will open a new PR to add comments and test cases for `conv_transpose` after the following PRs are merged.

- https://github.com/pytorch/pytorch/pull/97125
- https://github.com/pytorch/pytorch/pull/97126

Reply:
PR is created. Please take a look. Thanks! https://github.com/pytorch/pytorch/pull/97311